packages/react-table/src/components/Table/examples/Table.md

```diff
@@ -1994,6 +1994,166 @@ class FavoritesTable extends React.Component {
 }
 ```
 
+### Tree table
+To enable a tree table:
+1. Pass the `isTreeTable` prop to the `Table` component
+2. Add `role="tree"` to the `TableBody` component
+3. Pass the following props to each row:
+    - `isExpanded` - Flag indicating the node is expanded and its children are visible
+    - `isHidden` - Flag indicating the node's parent is expanded and this node is visible
+    - `level` - number representing how many levels deep this node is nested
+    - `posinset` - number representing where in the order this node sits amongst its siblings 
+    - `setsize` - number representing the number of children this node has
```

Comment:
I know this is an example, but would prefer to see `aria-posinset` and `aria-setsize` here, mainly because there is nothing to indicate these are aria attributes.